### DISCUSSION TITLE
Be in the results when looking up "Whatsapp" in gnome

### DISCUSSION THREAD

**Discussion by VichoReyes** [cstayyab/WALC#145] [Feature Requests]
When I want to open walc, I search my app menu for whatsapp most of the time, there are no results, and then I remember that the client has another name.
This could be improved by adding a comment field to the `WALC.desktop` file that says something like "Unofficial WhatsApp client" or similar, because gnome uses the comment field too when searching for applications.

**Comment by madroots** [comment]
I am using gnome and I can tell it finds WALC just fine when I look up "Whatsapp".

How did you install WALC in the first place? 

snap installation here and works great

**Comment by VichoReyes** [comment]
I used the AppImage
